newsfragments/539.bugfix

```diff
@@ -0,0 +1,6 @@
+This removes two unstated assumptions about the way goniometer axis metadata must be formatted when constructing a multi-axis diffractometer model from CBF image files.
+Firstly, it had been assumed that goniometer axes in the ``axis`` category would be listed in order of dependency, starting with the axis mounted on the lab floor and ending with the axis on which the sample is mounted (e.g. the order *ω*, *κ*, *φ* for a *κ*-geometry diffractometer).
+Separately, there was an assumption that the rotation angle metadata in the ``diffrn_scan_axis`` category would list the axes in the same order as in the ``axis`` category.
+The result was that perfectly valid data would be misinterpreted either because ``diffrn_scan_axis`` was ordered differently to ``axis``, or because the ``axis`` metadata were not in order of axis dependency.
```

Comment:
s/would/may be misinterpreted

Reply:
In the changelog, I'm referring to the issue in the past tense, so perhaps `s/would be/may have been/`.  Except that I'm specifically describing the cases where the misinterpretation _would_ happen, so there's no uncertainty about it.

Reply:
Ah, but I would agree with `s/because/when/g`.